schema/pgsql/schema.sql

```diff
@@ -105,32 +134,47 @@ CREATE TABLE rotation (
 
     -- A date in the format 'YYYY-MM-DD' when the first handoff should happen.
     -- It is a string as handoffs are restricted to happen only once per day
-    first_handoff date NOT NULL,
+    first_handoff date,
 
     -- Set to the actual time of the first handoff.
     -- If this is in the past during creation of the rotation, it is set to the creation time.
     -- Used by Web to avoid showing shifts that never happened
     actual_handoff bigint NOT NULL,
 
-    -- each schedule can only have one rotation with a given priority starting at a given date
+    changed_at bigint NOT NULL DEFAULT EXTRACT(EPOCH FROM NOW()) * 1000,
+    deleted boolenum NOT NULL DEFAULT 'n',
+
+    -- Each schedule can only have one rotation with a given priority starting at a given date.
+    -- Column must be NULLed for deletion via "deleted = 'y'".
     UNIQUE (schedule_id, priority, first_handoff),
```

Comment:
> Column must be NULLed

It's unclear which column this refers to. Also, this information is given inconsistently throughout the file, sometimes it's a comment on the constraint, sometimes on the column. Maybe this can be unified as well.

Reply:
I have unified the comments, put them above the `UNIQUE` line describing the reason and which columns must be NULLed.